src/compiler/tracing.ts

```diff
@@ -46,7 +46,7 @@ namespace ts.tracing {
         }
 
         const countPart =
-            mode === Mode.Build ? `.${++traceCount}` :
+            mode === Mode.Build ? `.${process.pid}.${++traceCount}` :

```

Comment:
Really picky, so feel free to ignore: I think that a dash makes more sense between the two...  (Makes it easier for mass processing of dump files, whenever that's needed.)

Reply:
I'm missing something - how would use a dash help with batch processing?

Reply:
Makes regexps a little bit easier: `\.(.*)\.json$` rather than a conditional `.`...

Reply:
I still don't get it, but I've switched to a dash.

Reply:
The above, vs `\.([^.]+(?:\.[^.]+\)?\.json` --- not usually needed, since the default operators are greedy, but even with that some uses will end up with the hairy regexp.